### ISSUE TITLE
Hide more than 500 chars post in TL

### ISSUE THREAD

**Issue by Technowix** [tootsuite/mastodon#5594]
When someone send a post with like, 5000 chars, it display the whole thing into Masto's UI and... take endless time to scroll.

So, might be cool to hide the >500 char from timeline, and make the "full post" readable when you click on it, so then everyone is happy !

* * * *

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
- [x] This bug happens on a [tagged release](https://github.com/tootsuite/mastodon/releases) and not on `master` (If you're a user, don't worry about this).

**Comment by RX14** [comment]
Perhaps the limit could be a multiplier (1.5x) of the default instance character limit. Or perhaps just something like 750 chars.

**Comment by Technowix** [comment]
Char limit is well "designed" to don't take too much time reading each posts & have bunch of space to write (and make old twitter users feel easy)

**Comment by egypturnash** [comment]
From the other end:

I have a multi-thousand-character toot limit, and I also don't want to have my instance blocked by other admins for being overly verbose. So I [force my users to put in a CW for toots over a sensible length](https://github.com/egypturnash/mastodon/commit/b9586c79629756dd0332b17f74a3bd13dae39a3f).

**Comment by Technowix** [comment]
@egypturnash smart!
almost what I wanted, but with a "read more" rather than a "content warning" ^^

**Comment by scroom** [comment]
@egypturnash Wow. Would be cool to have this as an option in the main branch!

**Comment by Cassolotl** [comment]
I think this might be a duplicate of *Hide bulk of longer posts from different sites #255*.
